searchlib/src/vespa/searchlib/features/global_sequence_feature.cpp

```diff
@@ -0,0 +1,64 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#include "global_sequence_feature.h"
+#include <vespa/vespalib/util/stash.h>
+
+using namespace search::fef;
+
+namespace search::features {
+
+namespace {
+
+/**
+ * Implements the executor for combining lid and distribution key to form a globally unique value.
+ */
+class GlobalSequenceExecutor : public fef::FeatureExecutor {
+private:
+    uint32_t _distributionKey;
+
+public:
+    GlobalSequenceExecutor(uint32_t distributionKey)
+        : _distributionKey(distributionKey)
+    {
+    }
+
+    void execute(uint32_t docId) override {
+        outputs().set_number(0, ((1ul << 48u) - ((uint64_t(docId) << 16u) | _distributionKey)));
+    }
+};
+
+}
+
+GlobalSequenceBlueprint::GlobalSequenceBlueprint() :
+    Blueprint("globalsequence"),
```

Comment:
consider camel case to match naming of other features

Reply:
Yes, the name should use came case.

Reply:
globalsequence -> globalSequence